lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -41,7 +41,7 @@ def cluster_attributes cluster, datacenter_name
 
         def cluster_path(cluster, datacenter_name)
           datacenter = find_raw_datacenter(datacenter_name)
-          cluster.pretty_path.gsub(/(#{datacenter.name}|#{datacenter.hostFolder.name})\//,'')
+          cluster.pretty_path.gsub(/(#{datacenter_name}|#{datacenter.hostFolder.name})\//,'')
         end
       end
```

Comment:
Got it. I guess those names are just not helping. `cluster_path` should be `cluster_relative_path` or something like that to be more explicit. Don´t need to comment on `pretty_path`. heheheh

Sorry to bother you but since i don´t have knowledge of vsphere i need some guidance. =)

This change seems pretty safe to me. Just to be sure, is there any place where `cluster_path` is used that might be expecting the behaviour you just mentioned is wrong?